snapm/manager/plugins/lvm2.py

```
@@ -195,6 +195,50 @@
+                    in_block = True
+                    continue
+                if not in_block or not line:
+                    continue
+                parts = line.split(maxsplit=1)
+                if len(parts) != 2:
+                    continue
+                major_str, name = parts
+                if name.strip() == "device-mapper":
+                    try:
+                        major = int(major_str)
+                    except (ValuerError, TypeError) as err:
```

Ruff (0.12.2)

Undefined name `ValuerError`

(F821)